functions/wav_time.h

```diff
@@ -0,0 +1,143 @@
+#ifndef FUNCTIONS_WAV_TIME_H
+#define FUNCTIONS_WAV_TIME_H
+
+#include "wav_effect.h"
+#include "int.h"
+#include "../blades/blade_base.h"
+
+// Usage: WavLen<EFFECT_NAME>  
+// EFFECT_NAMENUM : any off the available effect form the style edittor
+// Returns length of wav file returned as const int in [ms]
+// returned value: INTEGER
+//
+template<BladeEffectType EFFECT>
+class WavLen {
+ public:
+  void run(BladeBase* blade) {
+    found_ = false;
+    eff_name_ = wav_effect.GetEffectName(EFFECT);//find correspondig SoundEffectName for BladeEffectType
+    for (size_t i = 0; i < NELEM(wav_players); i++) {
+        if(strstr(eff_name_, wav_players[i].effectname()) && wav_players[i].isPlaying()) {found_ = true; idx=(int)i;}
+        if(strcmp(eff_name_,"pstoff")==0) {if(strstr(eff_name_, wav_players[i].effect2ndname()) && wav_players[i].isPlaying()) {found_ = true; idx=(int)i;}}
+            if (found_) { value_ = clampi32(wav_players[idx].length()*1000, 0, 32768);}
+            if (found_ && !shown_) {
```

Comment:
Do we really need to print things out other than for debugging?
It uses up FLASH memory for something that most users never see.
Also, if we're going to print something outside of commands, we should use default_output instead of STDOUT.

Reply:
Agreed for dveveloping and debugging this is very usefull, but does not need to be active once the final version is ready (or maybe via #define ENABLE_DEVELOPER_COMMANDS?)

STDOUT is used a lot throughout ProffieOS, so I figured it wouldn't hurt to use is. But if preference is to use default_output instead, that is an easy fix.